galgebra/ga.py

```diff
@@ -820,7 +823,7 @@ def _build_bases(self):
 
         self.mv_blades_lst = []
         for obj in self.blades_lst:
-            self.mv_blades_lst.append(self.mv(obj))
+            self.mv_blades_lst.append(mv.Mv(obj, ga=self))
```

Comment:
`self.mv` no longer works here, because `self.mv` calls `Mv.setup` which constructs both the requested vector and a new unique `mv_I` (???).


I'll look at gutting `Ga.mv` in a future patch.

Reply:
We need to be careful not to introduce potential recursion. I don't know if it jumps to too high an level which might then call back down to this method.(hard to describe the feeling when I only read this fractal piece of code, hope you get it)

Reply:
We're safe from recursion here - the danger I was avoiding was calling high-level methods on GA before __init__ is complete. This change ensures that.